sql-cli/tests/test___main__.py

```diff
@@ -1,5 +1,6 @@
 import pathlib
```

Comment:
As part of the urgency of having an E2E, we didn't test the features as we should, and a side-effect is that the test coverage went significantly down. We should address this ASAP.